go/test/endtoend/vtgate/queries/random/query_gen_test.go

```diff
@@ -0,0 +1,511 @@
+/*
```

Comment:
I'm thinking that this file does not have to have the `_test.go` ending. wdyt about just naming it `query_gen.go`?

Reply:
Maybe you can add a main function to generate and output your random queries instead of relying on a test function.